src/managed/Microsoft.Extensions.DependencyModel/Microsoft.Extensions.DependencyModel.csproj

```diff
@@ -39,6 +43,7 @@
     <Otherwise>
       <ItemGroup>
         <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+        <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.0.0" />
```

Comment:
Why was this version selected (2.5 years old) -- because it supports older TFMs?

Reply:
Both of these versions are what is used in `PlatformAbstractions` today, which we were getting through transitive dependencies:

https://www.nuget.org/packages/Microsoft.DotNet.PlatformAbstractions/3.0.0-preview-27324-5

```
System.AppContext (>= 4.1.0)
System.Collections (>= 4.0.11)
System.IO (>= 4.1.0)
System.IO.FileSystem (>= 4.0.1)
System.Reflection.TypeExtensions (>= 4.1.0)
System.Runtime.Extensions (>= 4.1.0)
System.Runtime.InteropServices (>= 4.1.0)
System.Runtime.InteropServices.RuntimeInformation (>= 4.0.0)
```

I am keeping these versions the same as they were before my change.